service/src/main/java/com/derpgroup/derpwizard/dao/impl/DynamoEmbeddedUserPreferencesDAO.java

```diff
@@ -0,0 +1,62 @@
+package com.derpgroup.derpwizard.dao.impl;
+
+import java.io.IOException;
+/*import java.util.ArrayList;
```

Comment:
That's incomplete, but it's just one of three implementations of that interface. It's meant for unit testing, but we already having a working implementation for unit testing (H2EmbeddedWhatever) - I just plan to eventually replace it with this DynamoEmbedded thing. The production class is complete (enough), though.